src/serviceregistry/manual/examples.md

```diff
@@ -0,0 +1,2356 @@
+              "accessAddresses": [
+                "192.168.100.100"
+              ],
+              "basePath": "/celsius-info",
+              "accessPort": 4040
+            }
+          }
+        ],
+        "createdAt": "2024-10-24T23:47:51Z",
+        "updatedAt": "2024-10-24T23:47:51Z"
+      },
+      {
+        "instanceId": "main-temperature-provider::celsius-info::1.0.0",
+        "provider": {
+          "name": "main-temperature-provider",
+          "metadata": {
+            "type": "temperature",
+            "customizable": true,
+            "scales": [
+              "Fahrenheit",
+              "Kelvin",
+              "Celsius"
+            ]
+          },
+          "version": "2.0.0",
+          "addresses": [
+            {
+              "type": "IPV4",
+              "address": "192.168.1.2"
+            },
+            {
+              "type": "HOSTNAME",
+              "address": "mtp.greenhouse.com"
+            }
+          ],
+          "device": {
+            "name": "thermometer5",
+            "metadata": {
+              "scales": [
+                "Fahrenheit",
+                "Kelvin",
+                "Celsius"
+              ],
+              "max-temperature": {
+                "Kelvin": 310,
+                "Celsius": 40,
+                "Fahrenheit": 140
+              },
+              "min-temperature": {
+                "Fahrenheit": 20,
+                "Kelvin": 260,
+                "Celsius": -10
+              },
+              "appliable": [
+                "spring",
+                "summer",
+                "autumn",
+                "winter"
+              ]
+            },
+            "addresses": [
+              {
+                "type": "MAC",
+                "address": "81:ef:1a:44:7b:02"
+              }
+            ],
+            "createdAt": "2024-10-22T10:34:10Z",
+            "updatedAt": "2024-10-22T10:34:10Z"
+          },
+          "createdAt": "2024-10-23T21:43:35Z",
+          "updatedAt": "2024-10-23T21:43:35Z"
+        },
+        "serviceDefinition": {
+          "name": "celsius-info",
+          "createdAt": "2024-10-24T21:48:36Z",
+          "updatedAt": "2024-10-24T21:48:36Z"
+        },
+        "version": "1.0.0",
+        "expiresAt": "2035-10-11T14:30:00Z",
+        "metadata": {
+          "frequency": 350
+        },
+        "interfaces": [
+          {
+            "templateName": "generic-http",
+            "protocol": "http",
+            "policy": "NONE",
+            "properties": {
+              "accessAddresses": [
+                "192.168.0.14"
+              ],
+              "accessPort": 4040,
+              "operations": {
+                "discover": {
+                  "path": "/discover",
+                  "method": "GET"
+                },
+                "query": {
+                  "path": "/query",
+                  "method": "POST"
+                }
+               },
+              "basePath": "/info/fahrenheit"
```

Comment:
something wrong here. This instance should be provide the celsius-info service.

Reply:
This is still wrong.  InstanceID: main-temperature-provider::celsius-info::1.0.0, but one of the interfaces (http) has a basePath /info/fahrenheit

Reply:
Also in the 3rd entry, the opposite case. InstanceID: main-temperature-provider::fahrenheit-info::1.0.0, but the basePath is /info/celsius